src/components/ItemFilters/ItemFilters.tsx

```diff
@@ -106,7 +111,14 @@ const ItemFilters = ({
 
   const handleYearSubmit = async (e: SyntheticEvent) => {
     e.preventDefault()
-    await submitFilters(year.length ? [year] : [], "year")
+    const isNumeric: boolean = /^[0-9]*$/.test(year)
+
+    if (year.length !== 4 || !isNumeric) {
```

Comment:
Double-check this. @charmingduchess I vaguely remember that there's a need for searching 3 digit years, right?

Reply:
We are currently validating for 4 characters in DFE, but it makes sense to me that 3 digit years (or fewer) would be valid.